### PR TITLE
Update README & owner for PRs

### PR DESCRIPTION
**Pull request by adipatel**
- Updating README to contain some build information and reference to the template action. 
- Adding CodeOwners & Reviewers.

### CODE REVIEW

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Set to true to add reviewers to pull requests
```
Comment (maddixa) [inline .github/auto_assign.yml:1]: Should we call it `autoassign-reviewers-workflow.yml` (also in the other repositories)? There is some logic in how github actions are used: they are part of a workflow https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow

File: README.md [hunk 2]
```diff
@@ -20,4 +20,31 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         teams: "my-github-team"         # only works for GitHub Organisation/Teams
         persons: "aditya"               # add individual persons here
```
Comment (maddixa) [inline README.md]:
```
          teams: "team1"
          persons: "???"
```

What is the format of persons: github-username or @github-username?

File: README.md [hunk 3]
```diff
@@ -20,4 +20,31 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (maddixa) [inline README.md]: `dixahq/autoassign-reviewers-action`

We need to (roughly) describe what rights are expected from the user token:

`${{ secrets.USER_TOKEN_THAT_CAN_ASSIGN_REVIEWERS_ACTION }}`

File: .github/auto_assign.yml (hunk 1 above)
Comment (adipatel) [reply .github/auto_assign.yml:1]: Sure, sounds like a good idea 👍 
Will update other repositories.

File: README.md (hunk 2 above)
Comment (adipatel) [reply README.md]: Will modify

File: README.md (hunk 3 above)
Comment (adipatel) [reply README.md]: Will add a comment about the token permission.

File: .github/auto_assign.yml (hunk 1 above)
Comment (adipatel) [reply .github/auto_assign.yml:1]: will update other repos. this is for assigning reviewers using our existing github app.

**Review by michidevv — APPROVED** [review]

**Pull request merged**